assembly/src/parsers/field_ops.rs

```diff
@@ -126,27 +126,25 @@ pub fn parse_inv(span_ops: &mut Vec<Operation>, op: &Token) -> Result<(), Assemb
     Ok(())
 }
 
-/// Translates pow2 assembly instruction to VM operations.
+/// Translates pow2 assembly instructions to VM operations.
 ///
 /// Specifically, we extract the least significant bit of the head element in the stack &
 /// perform a power of 2 operation on these individual bits. These individual powers are
-/// later aggregated to compute the power of 2 of the top value of the stack. In `unsafe mode`,
-/// we skip the check of verifying that the top element is less than 64 or not.
+/// later aggregated to compute the power of 2 of the top value of the stack. In the unchecked
+/// mode, we skip the check of verifying that the top element is less than 64.
```

Comment:
nit: I would update the beginning of this comment as well. I didn't understand what it was saying until I saw the edited comment below for `append_pow2_op`.

Reply:
Updated!